src/clientStubFromSlash.ts

```diff
@@ -3,6 +3,11 @@ import * as Url from "url-parse";
 import { DgraphClientStub } from "./clientStub";
 
 const PORT = "443";
+/**
+ * @deprecated since v21.3 and will be removed in v21.07 release. For more details, see:
+ *     https://discuss.dgraph.io/t/regarding-slash-cloud-dgraph-endpoints-in-the-clients/13492
```

Comment:
@vardhanapoorv This is an internal link, won't be visible to public. Should we make it public or remove this link?

Reply:
Oh right didn't know, @NamanJain8 shared it. I think we can make it public.